iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/impl/ExecFormImpl.java

```diff
@@ -45,6 +48,9 @@ public List<Tree> children() {
     result.add(leftBracket);
     result.addAll(argumentsWithSeparators.elementsAndSeparators());
     result.add(rightBracket);
+    if (garbage != null) {
+      result.add(garbage);
```

Comment:
I wonder if we should add it here. Would this mean that we will highlight this part of the instruction even if it's ignored by both Docker and us?

Reply:
I didn't add it at the beginning, but then it failed other parts of the parser.
The way our parser works requires us to provide in children all elements that leads to/is a `SyntaxToken`.